i18n-ui/src/Views/Sample/index.js

```diff
@@ -16,8 +16,7 @@ import {
 	TextField
 } from "@material-ui/core";
 import 'react-dropdown/style.css';
-import "../../index.css";
-import {TextArea} from "semantic-ui-react";
+import styles from "./styles.module.css";
```

Comment:
Remove it because not needed.
You need to create custom css with makeStyles from "@material-ui/core"

Reply:
fixed with !important in the module.css